src/chains/ethereum/ethereum/src/transaction-pool.ts

```diff
@@ -453,6 +455,33 @@ export default class TransactionPool extends Emittery<{ drain: undefined }> {
     return null;
   }
 
```

Comment:
I think we'll need a js-doc comment here. Could you add something like this? (feel free to refine)
```suggestion
  /**
   * Iterates over all transactions in each Heap for each key in the map.
   * Returns the transactions grouped by origin and nonce in the format: `{
   * origin1: { nonceA: ...txData, ... }, ...}`
   *
   * If `address` is specified, filters the result to only show transactions
   * from `address`
   * @param map
   * @param address
   * @returns Transactions grouped by origin and nonce.
   */
```